test/tests/bcs/coupled_dirichlet_bc/coupled_dirichlet_bc.i

```diff
@@ -0,0 +1,110 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 10
+  ny = 10
+[]
+
+[Variables]
+  [./u]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+
+  [./v]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+[]
+
+[Kernels]
+  [./diff_u]
+    type = Diffusion
+    variable = u
+  [../]
+
+  # Adding a CoupledForce term induces a coupling between the u and v
+  # equations.  Is this enough to trigger the off-diagonal Jacobian
+  # coupling?
+  [./coupled_force_u]
```

Comment:
Then - should we remove that from this test... or at least remove the comment?

The comment makes it sound like something is wrong...